src/tests/DBCleanup.test.ts

```diff
@@ -0,0 +1,37 @@
+import DBCleanup from "../services/DBCleanup";
+
+describe("DBCleanup", () => {
+  let cleanUpSpy: jest.SpyInstance;
+
+  beforeAll(() => {
+    // set to any because it is a private function
+    cleanUpSpy = jest
```

Comment:
because I wanted only to test if the constructor is working properly and calls this function in the right interval. I forgot to add tests for the cleanUp function itself. 
Will add them to this pull